src/panels/lovelace/components/hui-image.ts

```diff
@@ -66,11 +70,37 @@ export class HuiImage extends LitElement {
     if (this.cameraImage && this.cameraView !== "live") {
       this._startUpdateCameraInterval();
     }
+    this._intersectionObserver = new IntersectionObserver(
+      this.handleIntersectionCallback.bind(this),
+      {
+        threshold: [0.0],
+      }
+    );
+    this._intersectionObserver.observe(this);
   }
 
   public disconnectedCallback(): void {
     super.disconnectedCallback();
     this._stopUpdateCameraInterval();
+    if (this._intersectionObserver) {
+      this._intersectionObserver.disconnect();
+    }
+    this.imageVisible = undefined;
+  }
+
+  protected handleIntersectionCallback(entries) {
+    const wasImageVisible = this.imageVisible;
+    for (const entry of entries) {
```

Comment:
Since we only monitor 1 element with this observer, we can just get `entries[0].isIntersecting`

Reply:
fixed